.goreleaser.yml

```diff
@@ -12,7 +12,6 @@ builds:
     - darwin
   goarch:
     - amd64
```

Comment:
We can also add arm64 I think as you were mentioning and have two binaries releases

Reply:
IMO I would not have an arm64 build until you also have an arm64 docker image, which is out of scope for this PR.